src/backend/feed/processor.js

```diff
@@ -184,22 +184,28 @@ module.exports = async function processor(job) {
               ['pubDate', 'pubdate'],
               ['creator', 'author'],
               ['content:encoded', 'contentEncoded'],
+              ['updated', 'date'],
+              ['id', 'guid'],
+              ['media:group', 'mediaGroup'],
+              ['published', 'pubdate'],
             ],
           },
         },
         feed
       )
     );
-    const articles = await parser.parseURL(feed.url);
+
+    const feedXml = await parser.parseURL(feed.url);
```

Comment:
I think `articles` is still fine, since it's part of the Atom syndication spec.  This isn't really XML after it's parsed, so that's not a great name.

Reply:
I didn't know `articles` was a term of the specification. I changed it back to the previous name.